packages/@aws-cdk/aws-appsync/test/appsync.test.ts

```diff
@@ -2,5 +2,5 @@ import '@aws-cdk/assert/jest';
 import {} from '../lib';
 
 test('No tests are specified for this package', () => {
-  expect(true).toBe(true);
```

Comment:
any reason to change this?

Reply:
Just to pass `PR Linter / mandatory-changes (pull_request)` check.

I have no idea why there is only `expect(true).toBe(true);` here.

Reply:
Adding an integ-test will make this rule pass as well. Just leave the unit test as is, its just a placeholder, but no need to change it if we aren't gonna actually write something useful.